apps/site/lib/site_web/controllers/stop_controller.ex

```diff
@@ -480,4 +480,12 @@ defmodule SiteWeb.StopController do
 
     %{conn | assigns: assigns}
   end
+
+  def get_facilities(conn, %{"stop_id" => stop_id}) do
+    res =
+      [{"stop", stop_id}]
+      |> V3Api.Facilities.filter_by()
+
+    json(conn, res.data)
```

Comment:
**suggestion:** because `V3Api.Facilities.filter_by/1` can potentially return `{:error, any}`, we should check for that before assuming there's data to return.

Also please add unit tests for this function. :)